tomviz/PythonGeneratedDatasetReaction.cxx

```diff
@@ -370,10 +370,11 @@ void PythonGeneratedDatasetReaction::addDataset()
     layout->addWidget(buttons);
     dialog.setLayout(layout);
     dialog.layout()->setSizeConstraint(QLayout::SetFixedSize); //Make the UI non-resizeable
-
+    
```

Comment:
Blank lines should be empty with no spaces.  (I can fix this after you merge if you want though).